sycl/plugins/unified_runtime/ur/adapters/cuda/sampler.hpp

```diff
@@ -11,15 +11,24 @@
 /// Implementation of samplers for CUDA
 ///
 /// Sampler property layout:
-/// | 31 30 ... 6 5 |      4 3 2      |     1      |         0        |
-/// |      N/A      | addressing mode | fiter mode | normalize coords |
+/// |     <bits>     | <usage>
+/// -----------------------------------
+/// |  31 30 ... 6   | N/A
+/// |       5        | mip filter mode
+/// |     4 3 2      | addressing mode
+/// |       1        | filter mode
+/// |       0        | normalize coords
 struct ur_sampler_handle_t_ {
```

Comment:
Would it make sense to provide helpers for accessing the bitfields corresponding to different properties, so that we only keep the magic values in one place, the likes of:
`((hSampler->Props >> 2) & 7);`